config.cson

```diff
@@ -1857,6 +1862,11 @@ fileIcons:
 		match: ".dig"
 		colour: "medium-orange"
 		uses: "YAML"
+		
+	"Digital Command Language":
+        icon: "openvms"
+		match: ".com"
```

Comment:
The `.com` extension is already used by batch-scripts (identified using a Windows icon). Currently, `file-icons` doesn't support mapping multiple icons to the same file-extension (although it may in future).

Are there any other file-extensions (or filenames) we could use instead?